### ISSUE TITLE
Exclude test folder from composer package

### ISSUE THREAD

**Issue by xPaw** [FileEye/pel#11]
It contains a couple of megabytes of images, there's no need for them when installing the package with composer.

**Comment by diskerror** [comment]
Agreed, but only exclude the files when the flag "--no-dev" is used.
